src/php7/php_solr_document.c

```diff
@@ -1130,10 +1130,7 @@ PHP_METHOD(SolrDocument, sort)
 		RETURN_FALSE;
 	}
 
-	if (zend_hash_sort_ex(doc_entry->fields, zend_qsort, comparison_function, renumber TSRMLS_CC) == FAILURE) {
-
-		RETURN_FALSE;
-	}
+	zend_hash_sort_ex(doc_entry->fields, zend_qsort, comparison_function, renumber TSRMLS_CC);
```

Comment:
Simpler to use zend_hash_sort (zend_qsort being dropped in 8)
Other way in pr #22